This failed on MacOS with an error that seems related, but I don't quite follow what it wants:

```
error: failed to parse manifest at `/Users/runner/work/pgrx/pgrx/Cargo.toml`

Caused by:
  expected a boolean or an integer
  in `profile.dev.debug`
Error: Process completed with exit code 101.
```

Do we have different versions of rust between linux and mac in CI or does mac rust not support this value for `profile.dev.debug`?

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -431,8 +431,11 @@ jobs:
         # ls -lath `$(which pg_config) --pkglibdir` `$(which pg_config) --sharedir`/extension
         echo ""
 
-        echo "----- Output Cargo version -----"
-        cargo --version
+        echo "----- Stable Rust ----"
+        rustup update stable
+        rustup default stable
+        rustc -vV
+        cargo -vV
         echo ""
 
         echo "----- Outputting env -----"

